react/react1/week2/to-do-list/src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+import ToDoList from "./components/to-do-list";
```

Comment:
in React we use PascalCase not only to name the function components but also to name the files